ui/migrations/0010_video_default_sort.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-08-31 06:21
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ui', '0009_moira_lists'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='video',
+            options={'ordering': ['created_at']},
```

Comment:
Would descending order be better (more recent first)?  Maybe a question for Peter or Ferdi.

Reply:
right, more recent videos come first